plugin-ci/orb.yml

```diff
@@ -24,19 +24,36 @@ commands:
           from: << parameters.filename >>
           to: << parameters.bucket >>
           arguments: '--acl public-read --cache-control no-cache'
+  install-node:
+    description: Install node
+    steps:
+      - run:
+          name: Install Node
+          command: |
+            set +e
+            wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.1/install.sh | bash
+            export NVM_DIR="$HOME/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+
+            nvm install || nvm install 13.14 # use `.nvmrc` if it exists, otherwise use 13.14
+            nvm alias default $(node --version)
+
+            echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
 
   install-golangci-lint:
     description: Install golangci-lint
     parameters:
       version:
         type: string
-        default: 1.31.0
+        default: 1.46.2
```

Comment:
Should we go for https://github.com/golangci/golangci-lint/releases/tag/v1.49.0 directly?